prusti-tests/tests/verify/pass/loop-invs/nested-loops-3.rs

```diff
@@ -0,0 +1,41 @@
+use prusti_contracts::*;
+
+#[requires(n >= 0)]
+//#[ensures(result == old(n) * (1 + old(n) * (1 + old(n))))]
```

Comment:
Why is this (+ the invariants below) commented out?

Reply:
Good question. Maybe because Prusti was very slow with it, I'll check.

Reply:
It fails for some reason. I'll make a new test out of it.